jsettlers.logic/src/main/java/jsettlers/logic/map/loading/original/data/EOriginalMapSettlersType.java

```diff
@@ -33,7 +33,7 @@
 	SAWMILLER(EMovableType.SAWMILLER),
 	FORESTER(EMovableType.FORESTER),
 	BOWMAN_L1(EMovableType.BOWMAN_L1),
-	MELTER(EMovableType.MELTER),
+	IRON_MELTER(EMovableType.MELTER),
```

Comment:
is there also a GOLD_MELTER?

Reply:
No - this was erroneous, I reverted this change